library/nametab.mli

```diff
@@ -247,3 +247,8 @@ end
 
 module Make (U : UserName) (E : EqualityType) :
   NAMETREE with type user_name = U.t and type elt = E.t
+
+type modules_nametab
```

Comment:
Just a nit on style, maybe it'd be more convenient to use a module here.
```
module Modules : sig
  type t
  val freeze ...
  val unfreezze
end
```
At some point we may want to have a "Freezable" interface.

Reply:
Done.